### ISSUE TITLE
3 Punkte Menüsymbol neben Buch ohne Funktion

### ISSUE THREAD

**Issue by fb4thxUall** [jzakotnik/openlibry#119]
Siehe Titel. Stellt aber kein Problem dar.

**Comment by jzakotnik** [comment]
Ja, das ist noch für zukünftige Ideen da, was man machen könnte...

**Comment by fb4thxUall** [comment]
Ich schlage vor: Solange da noch nichts anderes ist, sollten die Punkte zur Detailseite des Buches führen. Das ist sonst unintuitiv. 

Bei der Gelegenheit: Auch der Klick auf das Cover eines Buches sollte zur Detailseite führen. 

LG

**Comment by fb4thxUall** [comment]
Danke. Auch wenn der Klick auf das Cover noch nicht zum Buch führt, ist das so schonmal eine gute Verbesserung. Ich schließe das hier erstmal.
